browser/admin/src/integrator/WordBook.ts

```diff
@@ -16,10 +16,110 @@ interface WordbookFile {
 	words: string[]; // list of dictionary words
 }
 
+const languageMapping: Record<string, string> = {
+	'<none>': 'All Language',
+	ar: 'Arabic',
```

Comment:
Is this not introducing a duplication? browser/src/control/Control.Zotero.js already has a language list, where each language is l10n in its own language. Also 'git grep Hungarian browser/' suggests we have translations for many language names in many languages already, which is not something you use here, it seems to me. Thanks.

Reply:
Hmm... So we have to pass this shorthand for the language to core (via wordbook). I noticed that while building core, we only supported these languages with shortcut names in distro-configs/CPLinux-LOKit.conf (line 66: --with-lang=ar bg ca cs da de el en-US en-GB eo es eu fi fr gl he hr hu id is it ja ko lo nb nl oc pl pt pt-BR sq ru sk sl sv tr uk vi zh-CN zh-TW). I was unable to find an exact list following this convention, so I created a new list here. I'm not sure if Wordbook allows passing any shorthand name, then maybe we can use it from Zotero.

cc: @caolanm @eszkadev